consul_awx_test.py

```diff
@@ -111,6 +129,12 @@ def test_mock(mocked_catalog_nodes, mocked_catalog_node):
                         "server_type": "postgresql",
                         "cluster": "one-two",
                     },
+                    "node5": {
+                        "ansible_host": "{}.0.0.4".format(ip_prefix),
+                        "datacenter": "dc1",
+                        "server_type": "postgresql",
+                        "cluster": 1,
```

Comment:
should this be a string?

Reply:
That value is formatted as an integer in get_host_vars
https://github.com/wilfriedroset/consul-awx/blob/master/consul_awx.py#L166-L167